README.md

```diff
@@ -250,7 +250,7 @@ Environment Variables
     -e "SECRET_ACCESS_KEY=secret_access_from_provider" \
     -e "ENDPOINT_URL=https://region.storage-provider.com" \
     -e "BUCKET_DIR=frappe-bench-v12" \
-    -v ./installation/sites:/home/frappe/frappe-bench/sites \
+    -v sites-vol:/home/frappe/frappe-bench/sites \
```

Comment:
```suggestion
    -v <project-name>_sites-vol:/home/frappe/frappe-bench/sites \
```

volumes generally have prefix when created as part of docker-compose

Reply:
Fixed in one other place as well.